public/app/features/explore/ResponsiveButton.tsx

```diff
@@ -1,40 +1,53 @@
-import React from 'react';
+import React, { forwardRef } from 'react';
 
 export enum IconSide {
   left = 'left',
   right = 'right',
 }
 
-type Props = {
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
This is kind of deceiving, we accept all HTMLDivElement props but do not actually pass them onto the div element just the two handlers needed for the tooltip. So if you added another handler or anything else it would be silently ignored. At this point I think wrapping it in div outside like:
```
<Tooltip>
  <div>
    <ResponsiveButton/>
  </div>
</Tooltip>
```
Would save us some work and complexity. As I said only think I would like to know is why the typing of tooltip did not error out before as it is typed in way that children need to be simple HTML element. If that worked this issue would be caught in compile time.